va_explorer/templates/va_data_management/index.html

```diff
@@ -46,7 +46,9 @@
         <td>{{ va.cause }}</td>
         <td>{{ va.warnings }}</td>
         <td>{{ va.errors }}</td>
-        <td><a class="btn btn-primary" href="{% url 'data_management:show' id=va.id %}">View</a></td>
+        {% if perms.va_data_management.view_verbalautopsy %}
```

Comment:
Do we actually need to check this here? The view itself is protected by a PermissionRequiredMixin so I don't think you could even get to index.html without alread having this permission.

Reply:
Great point! We don't need it here. I will remove it. :-)